bot/bot.go

```diff
@@ -43,8 +43,7 @@ func HandleMessageEvents(discord *discordgo.Session, msg *discordgo.MessageCreat
 		discord.ChannelMessageSend(msg.ChannelID, "https://imgur.com/K7lTDGU")
 	}
 
-	AddGriefer(discord, msg)
-	HandleCommands(discord, msg)
+	addGriefer(discord, msg)
```

Comment:
For the future it probably makes sense to register this as a separate event handler (like we did with CommandRegistry.HandleMessage [here](https://github.com/braydencw1/venova/pull/17/files#diff-8a7fed410ae8fe7d4deb7e8a06ef882c5e036ab39af60211c2fa433168f3ff15R50).